.github/workflows/playwright.yml

```diff
@@ -50,10 +50,13 @@ jobs:
       env:
         PLAYWRIGHT_TEST_BASE_URL: ${{ github.event.deployment_status.target_url }}
       run: yarn playwright test --shard=${{ matrix.shard }}
-    - name: Run Storybook tests
-      env:
-        TARGET_URL: '${{ github.event.deployment_status.target_url }}'
-      run: yarn test-storybook
+    - name: Build Storybook
+      run: yarn build-storybook --quiet
+    - name: Serve Storybook and run tests
+      run: |
+        npx concurrently -k -s first -n "SB,TEST" -c "magenta,blue" \
+          "npx http-server storybook-static --port 6006 --silent" \
+          "npx wait-on tcp:6006 && yarn test-storybook"
```

Comment:
@ninaTorgunakova 

To make the test suite easier to run locally, what would you think of moving the `yarn build-storybook` and commands to run the server and test suite into a script in the `scripts` directory, then adding a `yarn` workflow in `package.json` to execute the script? Then we could run the script here as well as locally by using a single command.

Otherwise, I think it's a little bit cumbersome to run the Storybook dev server, wait for it to be ready, then run `yarn test-storybook` in a separate terminal.

Reply:
That makes sense, yes, we can add this ability. But the execution of this single command can take a lot of time relative to local development. Would it be OK for you if this script takes about 25 seconds?

Reply:
It's not ideal, but I think it would make sense for making sure that local development corresponds with testing in CI. At 25 seconds, I can see myself developing manually using Storybook and the browser, then running the test script to make sure my changes don't break anything before pushing a feature branch.

Reply:
I added commands `storybook:test-local` and `storybook:test-ci` in [this commit](https://github.com/gravitational/docs/pull/312/commits/1e087e28b49209248254e8082566b80916e0c504).
They are mostly the same but `storybook:test-ci` includes `wait-on` command to ensure proper synchronization between the services and tests during the CI process which is unnecessary for local testing. 

With these commands, we can run tests locally/in CI using a single command without running Storybook dev server first. I added a description of them in `README.md` too.

I look forward to receiving feedback if this update becomes more convenient